packages/xo-server-sdn-controller/src/index.js

```diff
@@ -218,12 +245,18 @@ class SDNController extends EventEmitter {
         const networks = filter(xapi.objects.all, { $type: 'network' })
         const noVniNetworks = []
         await Promise.all(
-          map(networks, async network => {
-            if (network.other_config.private_pool_wide !== 'true') {
+          map(networks, async networkObj => {
+            // 2019-09-03
+            // Compatibility code, to be removed in 1 year.
+            await updateNetworkOtherConfig(networkObj)
+            const network = await networkObj.$xapi.barrier(networkObj.$ref)
```

Comment:
You should reassign directly the `networkObj` (you can name it `network` though), so this block of code can be removed without touching anything else.